### PR TITLE
chnages done in chiranjeevi file

### PR DESCRIPTION
**Pull request by Adi1122lakshmi**

### CODE REVIEW

File: chiranjeevi [hunk 1]
```diff
@@ -1 +1,2 @@
 first cinema 
+second cinema
```
Comment (Radhaprem) [inline chiranjeevi:2]: hi dude 
thanks for ur changes
i need more
or add another cinema

**Comment by Adi1122lakshmi** [comment]
reopen pull request

File: chiranjeevi [hunk 2]
```diff
@@ -1,2 +1,3 @@
 first cinema 
 second cinema
+third cinema
```
Comment (Radhaprem) [inline chiranjeevi:3]: tq for adding third line 
add fourth line also

Comment (Radhaprem) [reply chiranjeevi:3]: ok done

Comment (Adi1122lakshmi) [reply chiranjeevi:3]: cool can we resolve the commit radha

Comment (Radhaprem) [reply chiranjeevi:3]: ok lakshmi

**Pull request closed without merging**